src/Validator/Rules/DisableIntrospection.php

```diff
@@ -14,7 +14,7 @@ class DisableIntrospection extends QuerySecurityRule
     public const ENABLED = 1;
 
     /** @var bool */
-    private $isEnabled;
+    protected $isEnabled;
```

Comment:
```suggestion
    protected bool $isEnabled;
```

Generally, we should switch to fully typed properties and methods before opening up this class.

Reply:
I've typed all properties that changed from private to protected except for callable's.